internal/connector/internal/services/connector_namespaces.go

```diff
@@ -484,9 +485,8 @@ func (k *connectorNamespaceService) ReconcileDeletedNamespaces() (int, []*errors
 		if err := dbConn.Table("connector_namespaces").Select("connector_namespaces.id").
 			Joins("LEFT JOIN connector_statuses ON connector_statuses.namespace_id = connector_namespaces.id AND "+
 				"connector_statuses.deleted_at IS NULL").
-			Group("connector_namespaces.id").
```

Comment:
We also need this fix in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/bd75b56ea197b53e97fe1abbacad029c9f1ca361/internal/connector/internal/services/connector_cluster.go#L860